src/visitors/rules/structural/validateOpenAPIParameter.js

```diff
@@ -8,33 +8,52 @@ class ValidateOpenAPIParameter {
       name(node, ctx) {
         if (!node) return null;
         if (!node.name) return ctx.createError(ctx.messageHelpers.missingRequiredField('name'), 'key');
-        if (typeof node.name !== 'string') return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('string'), 'value');
+        if (typeof node.name !== 'string') {
+          return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('string'), 'value');
+        }
         return null;
       },
       in(node, ctx) {
         if (!node) return null;
         if (!node.in) return ctx.createError(ctx.messageHelpers.missingRequiredField('in'), 'key');
-        if (typeof node.in !== 'string') return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('string'), 'value');
-        if (!['query', 'header', 'path', 'cookie'].includes(node.in)) return ctx.createError("The 'in' field value can be only one of: 'query', 'header', 'path', 'cookie'", 'value');
+        if (typeof node.in !== 'string') {
+          return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('string'), 'value');
+        }
+        if (!['query', 'header', 'path', 'cookie'].includes(node.in)) {
+          return ctx.createError(
+            "The 'in' field value can be only one of: 'query', 'header', 'path', 'cookie'", 'value',
+          );
+        }
         return null;
       },
       description(node, ctx) {
-        if (node && node.description && typeof node.description !== 'string') return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('string'), 'value');
+        if (node && node.description && typeof node.description !== 'string') {
+          return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('string'), 'value');
+        }
         return null;
       },
       required(node, ctx) {
-        if (node && node.required && typeof node.required !== 'boolean') return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('boolean'), 'value');
+        if (node && node.required && typeof node.required !== 'boolean') {
+          return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('boolean'), 'value');
+        }
         if (node && node.in && node.in === 'path' && node.required !== true) {
-          return ctx.createError('If the parameter location is "path", this property is REQUIRED and its value MUST be true.', 'value');
+          return ctx.createError(
+            'If the parameter location is "path", this property is REQUIRED and its value MUST be true.',
```

Comment:
```suggestion
            'If the parameter location is "path", this property is required and its value must be true.',
```

The casing is inconsistent with required and must usage throughout.